src/modules/UI/scenes/ChangeMiningFee/components/CustomFees/CustomFeesModalConnector.js

```diff
@@ -10,9 +10,9 @@ import { updateMiningFees } from '../../../SendConfirmation/action'
 import { getCustomNetworkFee } from '../../../SendConfirmation/selectors.js'
 import { CLOSE_MODAL_VALUE } from '../../../WalletList/components/WalletOptions/action'
 import CustomFeesModal from './CustomFeesModal.ui'
-import type { CustomFees, CustomFeesModalOwnProps } from './CustomFeesModal.ui'
+import type { CustomFees } from './CustomFeesModal.ui'
 
-const mapStateToProps = (state: State, ownProps: CustomFeesModalOwnProps) => {
+const mapStateToProps = (state: State, ownProps: any) => {
```

Comment:
This is going backwards. Without an explanation, I wouldn't want to approve this.

Reply:
I submitted a commit that fixes this without using `any`. Please fix action.js below.